.gitignore

```diff
@@ -3,3 +3,4 @@
 *.rs.rustfmt
 target/
 Cargo.lock
+.idea/
```

Comment:
Yeah, that got inserted automatically for me; I had been ignoring it manually and must have missed it this time.

Just ignoring it across my entire computer is unfortunately a non-starter; [the canonical gitignore for IDEA](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore) is less restrictive than this and for my own repositories I don't mind spilling that little bit of IDE information in (basically, how to open the repository)).

I'll just keep manually ignoring; it's only 6 files and I can auto-git-add when creating the files, so it's just a mater of not accidentally adding them or letting the `.gitignore` get patched.